cf/mixin/propertiesdata.py

```diff
@@ -3457,6 +3457,36 @@ def file_locations(self):
 
         return set()
 
+    @_inplace_enabled(default=False)
+    def filled(self, fill_value=None, inplace=False):
+        """Replace masked elements with a fill value.
+
+        .. versionadded:: NEXTVERSION
+
+        :Parameters:
+
+            fill_value: scalar, optional
+                The fill value. By default the fill returned by
+                `get_fill_value` is used, or if this is not set then
```

Comment:
```suggestion
                `fill_value` is used, or if this is not set then
```

since this method isn't defined in the codebase so I guess from the below you mean this:

```pytb
>>> f.get_fill_value()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Field' object has no attribute 'get_fill_value'. Did you mean: 'fill_value'?
>>> print(f.fill_value())
None
```